wdl2cwl/main.py

```diff
@@ -373,26 +400,27 @@ def load_wdl_workflow(self, obj: WDL.Tree.Workflow) -> cwl.Workflow:
 
     def load_wdl_task(self, obj: WDL.Tree.Task) -> cwl.CommandLineTool:
         """Load task and convert to CWL."""
-        cwl_inputs = self.get_cwl_task_inputs(obj.inputs, obj.parameter_meta)
-        cwl_outputs = self.get_cwl_task_outputs(obj.outputs, obj.parameter_meta)
-        hints, requirements = self.get_cwl_hints_and_requirements(obj)
         description = obj.meta.pop("description", None)
-        if obj.meta:
-            _logger.warning("Skipping meta: %s", obj.meta)
+        cwl_inputs = self.get_cwl_task_inputs(obj.inputs, obj.parameter_meta)
         run_script = False
+        hints, requirements = self.get_cwl_hints_and_requirements(obj)
         for req in requirements:
             if isinstance(req, cwl.InitialWorkDirRequirement):
                 run_script = True
-        return cwl.CommandLineTool(
+        tool = cwl.CommandLineTool(
             id=obj.name,
             doc=description,
             inputs=cwl_inputs,
+            outputs=None,
             hints=hints,
             requirements=requirements,
-            outputs=cwl_outputs,
             cwlVersion="v1.2",
             baseCommand=["bash", "script.bash"] if run_script else ["true"],
         )
+        tool.outputs = self.set_cwl_task_outputs(obj.outputs, obj.parameter_meta, tool)
```

Comment:
so we can set `stdout` to a name of our choosing, if need be

Reply:
Oh, I thought it was weird to create the `tool` object, then pass it to that function. Then reading the code I saw the `glob_str = tool.stdout = "_stdout"`. Tried to think other ways to have it so we didn't need the function being called after the `tool` is created... but couldn't come up with anything not too messy :laughing: 

Thanks for adding the comment here :+1: